config/config.go

```diff
@@ -77,8 +102,10 @@ type Component struct {
 	AWSRegionBackend   *string           `json:"aws_region_backend"`
 	AWSRegionProvider  *string           `json:"aws_region_provider"`
 	AWSRegions         []string          `json:"aws_regions"`
+	EKS                *EKSConfig        `json:"eks,omitempty"`
```

Comment:
It would be nice if the arguments to the components were pluggable based on which kind of component it is, but not gonna block a 1st pass at this.